tests/logging/test_evaluation_tracker.py

```diff
@@ -126,9 +126,9 @@ def test_no_details_output(mock_evaluation_tracker: EvaluationTracker):
     assert not details_dir.exists()
 
 
-@pytest.mark.skip(  # skipif
+@pytest.mark.skipif(  # skipif
     reason="Secrets are not available in this environment",
-    # condition=os.getenv("HF_TEST_TOKEN") is None,
+    condition=os.getenv("HF_TEST_TOKEN") is None,
```

Comment:
I think the condition should be something else because it seems sometimes a token is there and not None but also not valid